RedditSharp/Multi/MultiData.cs

```diff
@@ -0,0 +1,183 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RedditSharp.Multi
+{
+    /// <summary>
+    /// Master Multi Class that contains the all of the Multi Data
+    /// </summary>
+    public class MultiData
+    {
+        /// <summary>
+        /// Kind of Multi
+        /// </summary>
+        [JsonProperty("kind")]
+        public string Kind { get; set; }
+
+        /// <summary>
+        /// Internal Model Data of the Multi Class
+        /// </summary>
+        [JsonIgnore]
+        public mData Data { get; set; }
+
+        /// <summary>
+        /// Creates an implementation of MultiData
+        /// </summary>
+        /// <param name="reddit">Reddit Object to use</param>
+        /// <param name="json">Json Token containing the information for the Multi</param>
+        /// <param name="webAgent">Web Agent to use</param>
+        /// <param name="subs">Whether there are subs</param>
+        protected internal MultiData(Reddit reddit, JToken json, IWebAgent webAgent, bool subs = true)
+        {
+            Data = new mData(reddit, json["data"], webAgent, subs);
+            JsonConvert.PopulateObject(json.ToString(), this, reddit.JsonSerializerSettings);
+        }
+    }
+
+    /// <summary>
+    /// Contains the innner information of the Multi
+    /// </summary>
+    public class mData
+    {
+        /// <summary>
+        /// Can the Multi be edited
+        /// </summary>
+        [JsonProperty("can_edit")]
+        public bool CanEdit { get; set; }
+
+        /// <summary>
+        /// Display name for the Multi
+        /// </summary>
+        [JsonProperty("display_name")]
+        public string DisplayName { get; set; }
+
+        /// <summary>
+        /// Actual name of the Multi
+        /// </summary>
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Description of the Multi in HTML format
+        /// </summary>
+        [JsonProperty("description_html")]
+        public string DescriptionHTML { get; set; }
+
+        /// <summary>
+        /// When the multi was created
+        /// </summary>
+        [JsonProperty("created")]
+        public double Created { get; set; }
```

Comment:
I'm assuming this is a date? May want to take a look at how the date parsing is done in some of the other objects and implement that here so it's easier to work with in C#